utils/travis/steps/install.sh

```diff
@@ -59,14 +59,6 @@ elif [ "$TRAVIS_OS_NAME" = "windows" ]; then
         ./utils/travis/windows-file-hasher.sh "projectfiles/$IMAGE/$CFG/filehashes.sqlite"
     fi
 else
-    # if not doing translations, save a bit of time by not copying them into the docker image
-    # otherwise, if this is the mingw job, the .git directory is needed for running the git archive command
-    if [ "$NLS" == "false" ]; then
-        echo "po/" >> .dockerignore
-    elif [ "$IMAGE" == "mingw" ]; then
-        rm .dockerignore
-    fi
```

Comment:
Was this intended to be part of this PR?

Reply:
Yes - it was a micro-optimization at best to improve the jobs' speed by ~30 seconds, but now that I also need `.git` in another job for this I'd rather not keep adding exceptions.